posts/snowflake-summit-2023-roundup.md

```diff
@@ -0,0 +1,75 @@
+
+Want to know more? Check out my blog on this session (here)[https://blog.ippon.tech/snowflake-summit-2023-replay-generative-ais-impact-on-enterprise-data-innovation/]!
+
+## Unified Iceberg Tables
+
+Apache Iceberg is a high performance open table format for large analytics datasets. With Iceberg you can bring the simplicity of SQL to big data wwhile allowing engines like Spark, Presto and Hibe to work with the same tables at the same time. In 2022, Snowflake partnered with Iceberg to combine Snowflake's unique capabilities with Apache Iceberg and Parquet to help customers solve some common challenges they face with big data: control, cost, and interoperability. This year, Snowflake is deepening the partnership by introducing Snowflake Iceberg Tables, which will extend Snowflake's performance and governance to data stored in open formats.
+
+Want to know more? Check out my blog on Unified Iceberg Tables (here)[https://blog.ippon.tech/snowflake-summit-2023-replay-iceberg-tables/]!
+
+## Platform Improvements
+
+I know, I know. It seems like we're taking a bit of a left turn from talking about all of the new announcements to Snowflake and the data processing industry as a whole. Just as important as the new features released, are the platform improvements Snowflake has been working on. During Summit this year, Snowflake announced a slew of platform updates, including the publich launch of the Snowflake Performance Index, the expansion of the Search Optimization Service (SO), and the introduction of TOP-K analytics.
+
+I detail all of the new Platform Performance updates (here)[https://blog.ippon.tech/snowflake-summit-2023-replay-snowflakes-put-the-customer-first-policy-improving-customers-price-for-performance/]!
+
+## Snowpark Container Services
+
+Another key announcement from Snowflake this week was the announcement of Snowpark Container Services. With Snowpark Container Services, you can now run data applications full stack in any language you'd like. The best part? You can package, distribute, and even monetize the applications you build in Snowflake by publishing them to the Snowflake Marketplace.
+
+I dive a bit deeper into Snowpark Container Services (here)[https://blog.ippon.tech/snowflake-summit-2023-replay-snowpark-container-development/]!
+
+## Document AI
+SSnowflake acquired Applica, a small company offering an AI-based text and document automation platform, late last year. Now, Snowflake has announced the launch of Document AI, now available in private preview. A new large language model (LLM), built from Applica's generative AI technology, Document AI, can help customers understand documents and put their unstructured data to work. Document AI allows you to ask questions in natural language from documents you have stored in Snowflake and operationalize an end-to-end pipeline in which you can provide feedback on the answers the model gives you to fine-tune and retrain it.
```

Comment:
Extra "S" at the beginning of this paragraph.  I'd also probably not describe Applica as a "small" company as it could potentially come across as "minimizing" the acquisition.